msm/skill_entry.py

```diff
@@ -227,7 +227,7 @@ def extract_repo_name(url):
 
     @staticmethod
     def extract_author(url):
-        return url.rstrip('/').split("/")[-2].split(':')[-1]
+        return url.split("/")[-2].split(':')[-1]
```

Comment:
Should probably be a rstrip here as well, fixes the failing unittest.

Reply:
Fixed. Sorry this got buried in my inbox